conf/cassandra.yaml

```diff
@@ -1375,6 +1375,30 @@ dynamic_snitch_reset_interval: 600000ms
 # until the pinned host was 20% worse than the fastest.
 dynamic_snitch_badness_threshold: 1.0
 
+# Configures Java crypto provider. By default, it will use DefaultCryptoProvider
+# which will install Amazon Correto Crypto Provider for x86 architecture.
+#
+# If you are on a different architecture (e.g. aarch), startup scripts will automatically
+# detect this and aarch version of the library is used.
+#
+# If the resolution of the architecture is not successful, or it somehow fails,
+# the crypto provider is not installed and default, in-built, crypto provider in your JRE
+# will be used.
+#
+# If you want to fail the startup of a node if the provider was not installed properly,
+# you have to set property "fail_on_missing_provider" to "true" which is by default "false".
+# This property is not specific for DefaultCryptoProvider, it will be present to every provider with same logic.
+
+# To install custom provider, you need to extend org.apache.cassandra.security.AbstractCryptoProvider,
+# put it to class path and reference here to use it.
+#
+# If you want to completely bypass the installation of a crypto provider, please use NoOpCryptoProvider.
```

Comment:
```suggestion
# Amazon Correto Crypto Provider works currently for x86_64 and aarch_64 platforms.  If this provider fails it will fallback to the default crypto provider in the JRE.
#
# To force failure when the provider was not installed properly, set the property "fail_on_missing_provider" to "true".
#
# To bypass the installation of a crypto provider use NoOpCryptoProvider.
```

my attempt and more concise and direct language.